recipes/vcftools/meta.yaml

```diff
@@ -1,27 +1,34 @@
+{% set version = "0.1.15" %}
+{% set sha256 = "bfbc50d92262868802d62f595c076b34f8e61a5ecaf48682328dad19defd3e7d" %}
+
+
 package:
   name: vcftools
-  version: '0.1.14'
+  version: {{ version }}
 
 source:
-  fn: vcftools-0.1.14.tar.gz
-  url: https://github.com/vcftools/vcftools/releases/download/v0.1.14/vcftools-0.1.14.tar.gz
-  md5: a110662535651caa6cc8c876216a9f77
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/vcftools/vcftools/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 5
+  number: 0
 
 requirements:
   build:
     - gcc  # [not osx]
     - llvm  # [osx]
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
     - perl
-    - perl-vcftools-vcf ==0.953
-
+    - perl-vcftools-vcf
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
   run:
     - libgcc  # [not osx]
-    - zlib
-    - perl-vcftools-vcf ==0.953
+    - zlib {{CONDA_ZLIB}}*
+    - perl-vcftools-vcf
```

Comment:
can you add `perl` as a dependency as well

Reply:
I'm wondering why the perl tools are added at all? Should this not be a separate package, for people that don't need perl.

Reply:
I don't know, as both C and perl libraries are included in the original package, people loading it would expect to have both available when using the bioconda package and discarding the perl libraries would cause confusion on the user's side no ? I didn't understand why they were split in the first place but I wanted to update the package following the previous work on bioconda ?